descriptors/ModuleDescriptor-template.json

```diff
@@ -222,26 +222,10 @@
       "id" : "login",
       "version" : "6.0 7.0"
     },
-    {
-      "id" : "service-points",
-      "version" : "2.1 3.0"
-    },
-    {
-      "id" : "service-points-users",
-      "version" : "1.0"
-    },
-    {
-      "id": "password-validator",
-      "version": "1.0"
-    },
     {
       "id": "authtoken",
       "version": "2.0"
     },
-    {
-      "id": "notify",
-      "version": "2.0"
-    },
```

Comment:
Based upon the current code in the mainline, when the notification request fails (anything other than a 201) an error is logged.

Is the intended behaviour of the system that when this interface is not provided?

Reply:
This is the interface for _sending_ notifications, isn't it?

Can you please point us to "the current code" that you mentioned?

Reply:
> This is the interface for _sending_ notifications, isn't it?

Yes. It is used to send password reset / confirmation e-mails to the user.

The password reset process _cannot_ work without being able to send notifications to the user, as the links in those e-mails are an essential part of the process.

> Can you please point us to "the current code" that you mentioned?

[This](https://github.com/folio-org/mod-users-bl/blob/e57f442047824e5e5b172981434af02181527b4b/src/main/java/org/folio/service/PasswordResetLinkServiceImpl.java#L167) is the code that triggers the sending of a notification to the user during a password reset.

[This](https://github.com/folio-org/mod-users-bl/blob/e57f442047824e5e5b172981434af02181527b4b/src/main/java/org/folio/rest/client/impl/NotificationClientImpl.java#L31-L33) is the code that interprets every non-201 response as a failure.